bentoml/serve.py

```diff
@@ -466,6 +476,10 @@ def serve_production(
                 numprocesses=api_workers or math.ceil(CpuResource.from_system()),
             )
         )
+
+        logger.info(
+            f'To interact with gRPC server using `gRPC UI`, use: "docker run -it --rm -p 8080:8080 fullstorydev/grpcui -plaintext localhost:{port}"'
+        )
```

Comment:
This wouldn't work in production.
You can have the same message logic like above.

Maybe we can refactor this into a function.